src/main/java/sirius/biz/storage/layer3/MutableVirtualFile.java

```diff
@@ -108,15 +108,25 @@ public static MutableVirtualFile unsafeCreate(@Nonnull VirtualFile parent, @Nonn
     /**
      * Provides an additional description which can be shown in the web based UI.
      *
-     * @param description the description to show
-     *                    (this will be {@link sirius.kernel.nls.NLS#smartGet(String) smart translated}).
+     * @param description the description to show (this will be {@link sirius.kernel.nls.NLS#smartGet(String) smart translated}).
```

Comment:
```suggestion
     * @param description the description to show (this will be {@linkplain sirius.kernel.nls.NLS#smartGet(String) smartly translated}).
```